irobot_benchmark/CMakeLists.txt

```diff
@@ -33,12 +33,89 @@ set(TOPOLOGY_FILES
   topology/debug_sierra_nevada_best_effort.json
   topology/debug_mont_blanc_reliable.json
   topology/debug_mont_blanc_best_effort.json
+  topology/debug_white_mountain_best_effort.json
 )
 
 install(FILES
   ${TOPOLOGY_FILES}
   DESTINATION lib/${PROJECT_NAME}/topology)
 
+set(SP topology/tests/single_process)
```

Comment:
What do you think about removing the `tests/` subdirectory?  The new topologies are already "identified" via the subdirectories that contain them, like `full_topologies`, `multi_process`, and `single_process`.

Reply:
LGTM!  

Another question though, the [white_mountain_fixed_size_debug.json](https://github.com/irobot-ros/ros2-performance/blob/mauro/test-ipc-loaned-3/irobot_benchmark/topology/full_topologies/white_mountain_fixed_size_debug.json) specifies `"qos_reliability":"best_effort"`.  But the analogous [sierra_nevada_fixed_size_debug.json](https://github.com/irobot-ros/ros2-performance/blob/mauro/test-ipc-loaned-3/irobot_benchmark/topology/full_topologies/sierra_nevada_fixed_size_debug.json) does not.  Intentional?

Reply:
I'm seeing a duplicate [debug_white_mountain_best_effort.json](https://github.com/irobot-ros/ros2-performance/blob/mauro/iron-events-executor/irobot_benchmark/topology/debug_white_mountain_best_effort.json) in the top-level `topology` directory.  But it is really (correctly, in my view) beneath the `full_topologies/` directory.  See
- [mauro/iron-events-executor branch](https://github.com/irobot-ros/ros2-performance/tree/mauro/iron-events-executor/irobot_benchmark/topology)
- [mauro/iron-to-galactic branch](https://github.com/irobot-ros/ros2-performance/tree/mauro/iron-to-galactic/irobot_benchmark/topology)

Reply:
Nice, just removed it.